apps/server/src/modules/authorization/domain/index.ts

```diff
@@ -0,0 +1 @@
+export * from './reference';
```

Comment:
Wieso hier der Ordner "domain"? Das ergibt für mich keinen Sinn. Eigentlich sollte doch der gesamte AuthorizationService Teil der Domain-Schicht sein. Aber speziell dieser Reference-Teil ist es doch gerade nicht, weil er die Repo-Schicht kennt.

Reply:
Für mich ist der Loader, der authorisation service und der auth ref service alles services. Daher Part des Domain Layers. 
Das die Services mit den Repo verbunden sind ist für mich ok. Allerdings steht auch hier aus das die Dependency Richtung sich ändern muss zwischen Repo und Service.

Reply:
Ja stimmt, wenn man da die Dependency umdrehen würde, dann wärs ja auch wie bei den anderen Services. Trotzdem macht für mich ein Ordner "domain" nur Sinn in einem Modul, in dem auch andere Layer vorhanden sind - als Unterscheidung zwischen den Layern. Aber hier ist doch das ganze Modul im Domain-Layer, oder nicht?

Reply:
Mhh stimmt auch wieder. Dann würde ich Domain weg lassen.

Reply:
Struktur geändert.

Reply:
Update: Ich bin noch am vollständigen ändern. Das zieht allerdings eine große Liste an File Änderungen nach sich die auch nest dependency cycles beinhalten. Evtl. ist die Änderung zu hart für diesen PR.

Reply:
Lösung gefunden, allerdings muss ich das jetzt noch überall einbauen.